### PR TITLE
added MAINTAINERS.md file

### PR DESCRIPTION
**Pull request by blu3beri**
Probably missing some maintainers. Feel free to add them.

Signed-off-by: blu3beri <berend@animo.id>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Maintainers
+
+### Active Maintainers
+| name         | Github                                         | Discord        |
+|--------------|------------------------------------------------|----------------|
+| Brent Zundel | [@brentzundel](https://github.com/brentzundel) | brent#0347     |
+| Mike Lodder  | [@mikelodder7](https://github.com/mikelodder7) | -              |
```
Comment (brentzundel) [inline MAINTAINERS.md:7]:
```suggestion
| Alexander Shcherbakov  | [@ashcherbakov](https://github.com/@ashcherbakov) | alexander.shcherbakov#5576              |
```

**Review by brentzundel — APPROVED** [review]
Mike has not been an active maintainer for some time. I'm happy to add him to the list

**Comment by mikelodder7** [comment]
I still review PRS I just haven’t submitted any new ones for a while

**Comment by brentzundel** [comment]
> I still review PRS I just haven’t submitted any new ones for a while

True, and if your name is still on the list our hopes that you'll return to us more fully will continue to burn brightly . . .

**Comment by brentzundel** [comment]
@mikelodder7 if you approve this PR we can officially keep you . . .

**Review by mikelodder7 — APPROVED** [review]

**Comment by mikelodder7** [comment]
Done

**Pull request merged**